src/utils/fetch-graphql.js

```diff
@@ -15,14 +16,14 @@ const handleErrorOptions = async ({
   variables,
   query,
   pluginOptions,
-  reporter,
+  structuredReporter,
 }) => {
   if (
     variables &&
     Object.keys(variables).length &&
     pluginOptions.debug.graphql.showQueryVarsOnError
   ) {
-    reporter.error(
+    structuredReporter.error(
```

Comment:
@sslotsky maybe we should have a debug code. In this case .error is being called but this is supplemental info to other errors that are occurring. It'd be good to be able to mark these as debugging errors maybe 🤔  what do you think?

Reply:
Sure, I can add a debug code!